app/models/user.rb

```
@@ -1121,20 +1121,43 @@ def should_subscribe_preview_be_regenerated?
     end
 
-    def cancel_active_subscriptions!
-      subscriptions.active.each { |s| s.cancel!(by_seller: false) }
-    end
+ 
 
-    def trigger_iffy_ingest
-      return unless saved_change_to_name? ||
-                    saved_change_to_username? ||
-                    saved_change_to_bio?
+  def cancel_active_subscriptions!
```

RuboCop (1.75.5)

[convention] Expected 1 empty line between method definitions; found 3.

(Layout/EmptyLineBetweenDefs)